src/cdi/CPEloss.hh

```diff
@@ -76,17 +78,18 @@ public:
   /*!
    * \brief Query to determine the target species type.
    */
-  virtual int32_t getTargetZAID() const = 0;
+  //virtual int32_t getTargetZAID() const = 0;
 
   /*!
    * \brief Query to determine the transporting particle type.
    */
-  virtual int32_t getProjectileZAID() const = 0;
+  //virtual int32_t getProjectileZAID() const = 0;
```

Comment:
Yes, for now the `CPEloss` class only provides the relevant `CParticle` object, which itself contains a ZAID. This can be refactored later if necessary.